Files already downloaded and verified
/ocean/projects/cis220031p/mmisra/miniconda3/envs/myenv/lib/python3.8/site-packages/torch/utils/data/dataloader.py:558: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 1, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
Traceback (most recent call last):
  File "/ocean/projects/cis220031p/mmisra/miniconda3/envs/myenv/lib/python3.8/multiprocessing/queues.py", line 245, in _feed
    send_bytes(obj)
  File "/ocean/projects/cis220031p/mmisra/miniconda3/envs/myenv/lib/python3.8/multiprocessing/connection.py", line 200, in send_bytes
    self._send_bytes(m[offset:offset + size])
  File "/ocean/projects/cis220031p/mmisra/miniconda3/envs/myenv/lib/python3.8/multiprocessing/connection.py", line 411, in _send_bytes
    self._send(header + buf)
  File "/ocean/projects/cis220031p/mmisra/miniconda3/envs/myenv/lib/python3.8/multiprocessing/connection.py", line 368, in _send
    n = write(self._handle, buf)
BrokenPipeError: [Errno 32] Broken pipe
epoch 0
epoch 1
epoch 2
epoch 3
epoch 4
4 {'recon_loss': 208.91336}
4 {'recon_loss': 210.7803}
epoch 5
epoch 6
epoch 7
epoch 8
epoch 9
9 {'recon_loss': 200.65182}
9 {'recon_loss': 204.74043}
epoch 10
epoch 11
epoch 12
epoch 13
epoch 14
14 {'recon_loss': 196.12505}
14 {'recon_loss': 204.13504}
epoch 15
epoch 16
epoch 17
epoch 18
epoch 19
19 {'recon_loss': 191.59148}
19 {'recon_loss': 202.69345}
